qemu/tests/cfg/migrate_nic_with_iommu.cfg

```diff
@@ -1,7 +1,11 @@
 - migrate_nic_with_iommu:
     virt_test_type = qemu
     only virtio_net
-    image_snapshot = yes
+    clone_master = yes
```

Comment:
So how to handle the issue you mentioned at https://github.com/autotest/tp-qemu/pull/3587

Reply:
> So how to handle the issue you mentioned at #3587

Oh, thanks for your reminder, I had forgot this patch mentioned issues. So I think it should be changed to the following lines, please correct me if I'm wrong:
```
backup_image_before_testing = yes
restore_image_after_testing = yes
```

Reply:
Sounds good to me.